### PR TITLE
toggle: update exported overridable id

### PR DESCRIPTION
**Pull request by jennur**
Closes https://github.com/inveniosoftware/react-invenio-deposit/issues/469

### CODE REVIEW

File: src/lib/components/Toggle/Toggle.js [hunk 1]
```diff
@@ -66,4 +66,4 @@ ToggleComponent.defaultProps = {
   overridableId: "",
 };
 
-export default Overridable.component("SearchFilters.Toggle", ToggleComponent);
+export default Overridable.component("SearchFilters.Toggle.element", ToggleComponent);
```
Comment (kpsherva) [inline src/lib/components/Toggle/Toggle.js:69]: wy is this change needed? it is not consistent with the rest of the components ids and potentially a breaking change

**Comment by jennur** [comment]
Should be updated in https://github.com/inveniosoftware/react-invenio-deposit/blob/master/src/lib/components/License/LicenseModal.js#L30 instead.
